hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RSRpcServices.java

```diff
@@ -983,6 +983,11 @@ private void checkCellSizeLimit(final HRegion r, final Mutation m) throws IOExce
           throw new DoNotRetryIOException(msg);
         }
       }
+    } else {
+      String msg = "Cell size check is disabled because of maxCellSize is set to" + r.maxCellSize;
+      if (LOG.isDebugEnabled()) {
+        LOG.debug(msg);
+      }
```

Comment:
We should use log4j parameterized messages feature, that way we don't need the concatenation logic, nor the if statement.

Reply:
ok